clusterloader2/pkg/measurement/common/ooms_tracker.go

```diff
@@ -213,6 +215,15 @@ func (m *clusterOOMsTrackerMeasurement) handleOOMEvent(_, obj interface{}) {
 		return
 	}
 
+	m.lock.Lock()
+	defer m.lock.Unlock()
+
+	if m.resourceVersionRecorded[event.ObjectMeta.ResourceVersion] {
+		klog.Warningf("caught an OOM event with already recorded resource version: %+v", event)
```

Comment:
Given that it may happen on the re-list, we shouldn't log this as warning. Warning means that something unexpected happened, it's not the case. Let's log as info or not log at all

Reply:
Done - I replaced logging with a comment containing a rephrasing of your remark ;).